e2e_tests/tests/fixtures/pytorch_identity/model_def.py

```diff
@@ -51,7 +51,7 @@ def evaluate_batch(self, batch: pytorch.TorchData) -> Dict[str, Any]:
         data, label = batch
 
         loss = self.loss_fn(self.model(data), label)
-        weight = self.model.weight.data.item()
+        weight = self.model.module.weight.data.item()
```

Comment:
does this work, even for non-ddp models?

Reply:
changed this test model to check depending on DDP or not.